lib/plugins/features/movies/index.js

```diff
@@ -0,0 +1,25 @@
+'use strict';
+
+const Controller     = require('./controller');
+const MovieValidator = require('../../../validators/movie');
+
+exports.register = (server, options, next) => {
+
+  server.route([{
+    method: 'POST',
+    path: '/movies',
```

Comment:
Ideally following the REST convention that we use should be enough to know what the endpoint does.

`POST /movies` creates movies
`DELETE /moves` deletes movies
`GET /movies/{id}` fetches a movie
`GET /movies` lists all movies (or some)